Gemfile.lock

```diff
@@ -32,7 +32,7 @@ GEM
     builder (3.2.2)
     capybara (2.3.0)
       mime-types (>= 1.16)
-      nokogiri (>= 1.3.3)
+      nokogiri (>= 1.6.4)
```

Comment:
Why did we need to update this version? I think we're locked on 1.3 for `reasons`